.github/workflows/rsc/dockerfile_lint_rules.yaml

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
There's something wrong with this file. When I run dockerfile_lint locally with this rule file specified it breaks:
```
dockerfile_lint -r .github/workflows/rsc/dockerfile_lint_rules.yaml Dockerfile 

# Analyzing Dockerfile

TypeError: Cannot use 'in' operator to search for 'FROM' in null
    at validateCommand (/usr/local/lib/node_modules/dockerfile_lint/lib/linter.js:39:21)
    at Linter.validate (/usr/local/lib/node_modules/dockerfile_lint/lib/linter.js:119:9)
    at lint (/usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:120:36)
    at /usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:129:13
    at new Promise (<anonymous>)
    at lintDockerFile (/usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:107:12)
    at /usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:86:16
    at Function.map (/usr/local/lib/node_modules/dockerfile_lint/node_modules/lodash/dist/lodash.js:3509:27)
    at lintDockerFiles (/usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:82:26)
    at Object.<anonymous> (/usr/local/lib/node_modules/dockerfile_lint/bin/dockerfile_lint:172:5)
```

And running it via statick also shows some errors:
```
INFO:root:Running dockerfile-lint tool plugin...
WARNING:root:ValueError: Expecting value: line 1 column 1 (char 0)
WARNING:root:ValueError: Unterminated string starting at: line 1924 column 20 (char 65530)
WARNING:root:ValueError: Expecting value: line 1 column 1 (char 0)
WARNING:root:ValueError: Unterminated string starting at: line 1924 column 20 (char 65530)
```

Reply:
Ah, good catch.  It's probably the whitespace corrections I did to make statick happy.  The indentations are messed up in the default file, and I must have missed some change I made to the operation.

Reply:
This seems to be also an issue with statick?  Because shouldn't it fail if a plugin fails in some way?  (I didn't catch this because statick was happily passing even though the plugin was failling).

Reply:
I'll look more closely, but Statick is meant to fail by default only if there is a problem with internal operations. The `--check` flag is used to tell Statick to output a failed return code if any issues are discovered or if any of the tools are not found.

Reply:
The `--check` flag is documented in Statick at https://github.com/sscpac/statick#concepts. That is a long README and I can see how that documentation is not obvious. I'd be happy to update the documentation if we can come up with a better way. Perhaps that paragraph should be a new subsection so that it shows up in the Contents.